docs/source/tutorials/node-audio-processing.mdx

```diff
@@ -0,0 +1,102 @@
+
+# Server-side Audio Processing in Node.js
+A major benefit of writing code for the web is that you can access the multitude of APIs that are available in modern browsers. Unfortunately, when writing server-side code, we are not afforded such luxury, so we have to find another way. In this tutorial, we will design a simple Node.js application that uses Transformers.js for speech recognition with Whisper, and in the process, learn how to process audio on the server.
+
+The main problem we need to solve is that the [Web Audio API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Audio_API) is not available in Node.js, meaning we can't use the [`AudioContext`](https://developer.mozilla.org/en-US/docs/Web/API/AudioContext) class to process audio. So, we will need to install third-party libraries to obtain the raw audio data. For this example, we will only consider `.wav` files, but the same principles apply to other audio formats.
+
+<Tip>
+
+This tutorial will be written as an ES module, but you can easily adapt it to use CommonJS instead. For more information, see the [node tutorial](https://huggingface.co/docs/transformers.js/tutorials/node).
+
+</Tip>
+
+
+**Useful links:**
+- [Source code](https://github.com/xenova/transformers.js/tree/main/examples/node-audio-processing)
+- [Documentation](https://huggingface.co/docs/transformers.js)
+
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/en/) version 16+
+- [npm](https://www.npmjs.com/) version 7+
+
+
+
+## Getting started
+
+Let's start by creating a new Node.js project and installing Transformers.js via [NPM](https://www.npmjs.com/package/@xenova/transformers):
+
+```bash
+npm init -y
+npm i @xenova/transformers
+```
+
+<Tip>
+
+Remember to add `"type": "module"` to your `package.json` to indicate that your project uses ECMAScript modules.
+
+</Tip>
+
+
+Next, let's install the [`wavefile`](https://www.npmjs.com/package/wavefile) package, which we will use for loading `.wav` files:
+
+```bash
+npm i wavefile
+```
+
+
+## Creating the application
+
+Start by creating a new file called `index.js`, which will be the entry point for our application. Let's also import the necessary modules:
```

Comment:
I wonder if we should just start using `index.mjs`? If people want to use CommonJS, they can leave it at `index.js` or if `type=module` is indicated in `package.json`, devs can use `index.cjs`. It just makes the tutorial shorter without going into these issues that only arise because we cling to `*.js` :thought_balloon:

Reply:
Linked to https://github.com/xenova/transformers.js/issues/152. I'll mark this comment as resolved, but will take note of this as we add support for CommonJS usage.